scripts/sdk_process/patch_models.py

```diff
@@ -36,6 +36,7 @@
 """
 
 track2_header = copyright_header + b"""import datetime
+import msrest
```

Comment:
Is this line necessary?

Reply:
Yes. I have no motivation to modify this file unless I meet an error.